README.md

```diff
@@ -32,27 +32,26 @@ Please see the [setup guide](SETUP.md) for more details on setting up your machi
 
 To setup on your local machine:
 
-1. Install Anaconda with Python >= 3.6. [Miniconda](https://conda.io/miniconda.html) is a quick way to get started.
-
-2. Clone the repository
+To install core utilities, CPU-based algorithms, and dependencies
 
+1. Ensure software required for compilation is installed. On Linux this can be supported by adding build-essential dependencies:
 ```bash
-git clone https://github.com/Microsoft/Recommenders
+sudo apt-get install -y build-essential
 ```
+On Windows you will need [Microsoft C++ Build Tools](https://visualstudio.microsoft.com/visual-cpp-build-tools/)
+  
```

Comment:
A question about this, build-essential is only needed in xlearn or is it needed also for the core deps?

Reply:
What I did here is I just copied the text from the setup guide. It was introduced here https://github.com/microsoft/recommenders/blob/b170c8459d91480d11ec450eb5e99e4f321af182/SETUP.md 
@gramhagen could confirm but this is not only about xlearn but it's more general. These are the build tools and libraries (like gcc, g++) that are needed for some pip packages to install. Normally they should be there (they are in the DSVM for example), but it's good to be up to date just in case. 
xlearn has the additional dpeendency on cmake. Cmake is not included in build-essential; it depends on it though.

Reply:
yeah, this is required by many of the libraries, not just xlearn. if you install pip through `apt-get install python3-pip` it will install `build-essential` by default since it is so common. but other methods of installing pip don't do that, so it's important to provide this step.